.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on: [push, 	pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest  # TODO specify disto and version
```

Comment:
Close all TODO`s before you try to merge something into the main branch

Reply:
done